app/filtering.js

```diff
@@ -489,9 +503,7 @@ function registerPermissionHandler (session, partition) {
         }
       }
 
-      // Display 'Brave Browser' if the origin is null; ex: when a mailto: link
-      // is opened in a new tab via right-click
```

Comment:
i took this out because it seems to no longer be needed. verified with this PR that if i go to brave.com and right-click the `mailto` links at the bottom of the page, they show a non-null origin.